src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -82,6 +82,8 @@ static ErrorFacts()
             nullableWarnings.Add(GetId(ErrorCode.WRN_NullabilityMismatchInReturnTypeOnInterceptor));
             nullableWarnings.Add(GetId(ErrorCode.WRN_NullabilityMismatchInParameterTypeOnInterceptor));
 
+            nullableWarnings.Add(GetId(ErrorCode.WRN_UninitializedNonNullableBackingField));
```

Comment:
>nullableWarnings [](http://example.com/codeflow?start=12&length=16)

Does this allow disabling the warning using `pragma warning disable nullable`? It doesn't look like we're testing the existing entries above, but consider adding a test for this case.
```csharp
#nullable enable
#pragma warning disable nullable
class C1
{
    object P1 => field; // 1
}
#pragma warning enable nullable
class C2
{
    object P2 => field;
}
```

Reply:
I think `#pragma warning disable nullable` was ripped out, but, it should work with `#nullable disable warnings`.